test/contracts/ContractInterface.h

```diff
@@ -67,8 +67,8 @@ class ContractInterface
 		bytes const& ret = call(_name + "(string)", u256(0x20), _arg.length(), _arg);
 		BOOST_REQUIRE(ret.size() == 0x20);
 		BOOST_CHECK(std::count(ret.begin(), ret.begin() + 12, 0) == 12);
-		// TODO: simplify this
-		return u160(u256(util::h256(ret)));
+		bytes const addr{ret.begin() + 12, ret.end()};
```

Comment:
Can also do the same below to get rid of `bytesConstRef` here. Since we are only copying 20 bytes in a test I see very little need for `bytesConstRef`.